code/modules/food_and_drinks/food/customizables.dm

```diff
@@ -1,43 +1,43 @@
+#define MAKE_CUSTOM_FOOD(snack_to_add, user, type) \
```

Comment:
Interesting choice to make this a define. I'm curious, did it noticeably help performance?

Reply:
Mostly made it since I didn't want to make a proc on item level to make a snack. Seeing that you can combine things on a plate (which ain't a snack). I made it mostly to reduce having to rewrite the same code multiple times :D

Reply:
Ah yeah, didn't notice one of the items was a `/obj/item/trash/plate`, while others are snacks, so that makes sense.